composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
+      "post-autoload-dump": [
+        "php -r \"symlink('../../../../../../vendor/swagger-api/swagger-ui', './src/Bridge/Symfony/Bundle/Resources/public/swagger-ui');\""
```

Comment:
You know, when you think about it, it's perfectly understandable because Composer simply clones the repository or download the archive from GitHub.